star_test.py

```diff
@@ -49,26 +49,28 @@ def example_star(starname=""):
 
         print(sun_like_star)
 
-    n = 0
     items = ["density", "temperature", "mass", "luminosity", "opticaldepth"]
     # This nixt line is needed for saving data to a text file
-    array2D = [[] for i in range(len(items))]
+    array2D = [[] for i in range(len(items) + 1)]
+    array2d[0] = radii_steps * step_size / 696342000, # Units of solar radius
+    n = 1
+
     for item in items:
         array2D[n] = sun_like_star.properties[item].val[0, :-1]
         plt.plot(
-            radii_steps * step_size / 696342000, # Units of solar radius
+            array2D[0], # Units of solar radius
             array2D[n], # Plots the value of item
             label=item)
         plt.xlabel("Solar Radii")
         plt.title(item)
-        plt.savefig("%s.png" % (item))
+        #plt.savefig("%s.png" % (item)) # Don't need because of plotting script
```

Comment:
```suggestion
        plt.show()
```
Is it worth showing and not saving? We can just remove all of the plotting stuff from this script if you think so.

Reply:
Just thought that it's not necessarily useful to save the figures because they can cause clutter, so we can just view the figures instead without saving and use the new plotting script to save stuff